eng/Versions.props

```diff
@@ -8,8 +8,8 @@
   <PropertyGroup Label="Version settings">
     <AspNetCoreMajorVersion>6</AspNetCoreMajorVersion>
     <AspNetCoreMinorVersion>0</AspNetCoreMinorVersion>
-    <AspNetCorePatchVersion>15</AspNetCorePatchVersion>
-    <ValidateBaseline>true</ValidateBaseline>
+    <AspNetCorePatchVersion>16</AspNetCorePatchVersion>
+    <ValidateBaseline>false</ValidateBaseline>
```

Comment:
Why is the validateBaseline changing?

Reply:
It has to be `false` between branding day & Patch Tuesday when we do the internal->public merge, because the baseline can't be updated until then. Sean's script auto-sets this property every month.